stdlib/runtime/numberUtils.gr

```diff
@@ -866,20 +879,31 @@ let utoa64_any_core = (buffer, num, offset, radix) => {
   let base = WasmI64.extendI32U(radix)
   let mut num = num
   let mut offset = offset
-  if ((radix & (radix - 1n)) == 0n) { // for radix which pow of two
+  if ((radix & radix - 1n) == 0n) { // for radix which pow of two
     let shift = WasmI64.extendI32U(WasmI32.ctz(radix) & 7n)
     let mask = WasmI64.sub(base, 1N)
-    for (;;) {
+    for (; ;) {
```

Comment:
We don't want this space, right? @ospencer

Reply:
I can explain.  It looked weird when we had just the middle clause empty.  I agree it looks wrong here though.